### DISCUSSION TITLE
PS2, esp32 pairs with controller but console not reading inputs. If I unplug and replug the controller cable then it works.

### DISCUSSION THREAD

**Discussion by klunkee** [darthcloud/BlueRetro#275] [Q&A]
I tried using a DS4 mainly but the same thing happens with a Wii classic controller. The controller pairs but no input is read unless I physically reconnect the cable on the console. I also tried pressing the reset button on the esp32 which does disconnect the controllers but they still don't respond after pairing. Is there anything I can do to get it to work without having to reconnect the cable each time?

**Comment by DJm00n** [comment]
May be related to #250. Fixed in my case after I uploaded psx firmware instead of generic one.
